classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students = student
+    student.classroom = self
+  end
```

Comment:
- [ ] I am highlighting this because no change in the `student` class please consider adding a method that serves as the `belongs-to` side (a student belongs to a classroom) in the `Student` class
```ruby
def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```

Reply:
- I suggest that you make a `classroom` an `attr_reader` instead of `attr_accessor` we only want to read the value, but not change it.